pkg/component/networking/policy/denyall/denyall.go

```diff
@@ -0,0 +1,93 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package denyall
```

Comment:
Why do we need a complete new component package and `ManagedResource` just to deploy a simple `NetworkPolicy`? For pragmatic reasons (i.e., to eliminate boilerplate as much as possible), please just put it into https://github.com/gardener/gardener/blob/master/pkg/component/shoot/system - this should be good enough.